app/src/main/java/com/moyerun/moyeorun_android/views/BadgeRoundImageView.kt

```diff
@@ -0,0 +1,110 @@
+package com.moyerun.moyeorun_android.views
+import android.Manifest
+import android.content.Context
+import android.content.res.TypedArray
+import android.util.AttributeSet
+import android.util.Patterns
+import android.view.LayoutInflater
+import androidx.annotation.ColorRes
+import androidx.annotation.Dimension
+import androidx.annotation.DrawableRes
+import androidx.annotation.RequiresPermission
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.bumptech.glide.Glide
+import com.moyerun.moyeorun_android.R
+import com.moyerun.moyeorun_android.databinding.ViewBadgeRoundImageBinding
+import java.io.IOException
+
+class BadgeRoundImageView @JvmOverloads constructor(
+    context: Context, attrs: AttributeSet?=null, defStyle: Int = 0)
+    : ConstraintLayout(context, attrs, defStyle) {
+    private var binding: ViewBadgeRoundImageBinding = ViewBadgeRoundImageBinding.inflate(LayoutInflater.from(context), this, true)
+
+    init {
+        if(attrs != null) { setAttrs(attrs, defStyle) }
+    }
+
+    override fun onMeasure(widthMeasureSpec: Int, heightMeasureSpec: Int) {
+        val verticalPadding = paddingTop + paddingBottom
+        val horizontalPadding = paddingStart + paddingEnd
+        if(widthMeasureSpec >= heightMeasureSpec) {
+            super.onMeasure(heightMeasureSpec + horizontalPadding, heightMeasureSpec + verticalPadding)
+        } else {
+            super.onMeasure(widthMeasureSpec + horizontalPadding, widthMeasureSpec + verticalPadding)
+        }
+    }
+
+    private fun getAttrs(attributeSet: AttributeSet, customViewAttrs: IntArray, defStyle: Int=0, defStyleRes: Int=0)
+            = context.obtainStyledAttributes(attributeSet, customViewAttrs, defStyle, defStyleRes)
+
+    private fun setAttrs(attrs: AttributeSet, defStyle: Int) {
+        val attributesTypedArray = getAttrs(attrs, R.styleable.BadgeRoundImageView, defStyle, 0)
+        setTypedArray(attributesTypedArray)
+    }
+    private fun setTypedArray(typedArray: TypedArray) {
+        binding.imgBigCircle.apply {
```

Comment:
apply 는 instance의 속성을 설정하고 instance를 반환하기 때문에 반환된 instance를 사용하지 않을 경우에는 with()를 사용하는 것이 좋아 보입니다.
```suggestion
        with (binding.imgBigCircle) {
```

Reply:
앗ㅎㅎ 좋은 지적 감사합니다..!!